adobe/react-spectrum
# fix(#4103): ProgressCircle: overBackground variant is using Canvas as fill color instead of Highlight in HCM

Since backgrounds should be removed or set to `Canvas` in HCM, we shouldn't need a dedicated style for overBackground.

Also, add forced-colors:active override for `--spectrum-loader-circle-medium-track-color`, so that the track color will be correct within components, like Picker that use forced-color-adjust: none.


Closes #4103 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 4103](https://github.com/adobe/react-spectrum/issues/4103).

## 📝 Test Instructions:

With Windows High Contrast Mode, or emulating forced-colors: active using dev tools.

1. Open https://reactspectrum.blob.core.windows.net/reactspectrum/b9f9bd8450ac5ff382206b6528b5aec12cf61471/storybook/index.html?path=/story/progress-progresscircle--isindeterminate-true-variant-overbackground&providerSwitcher-express=false
2. Verify that the ProgressCircle track within the Picker is using CanvasText color, and the fill is using Highlight color.
3. Open https://reactspectrum.blob.core.windows.net/reactspectrum/b9f9bd8450ac5ff382206b6528b5aec12cf61471/storybook/index.html?path=/story/picker--loading&providerSwitcher-express=false 
4. Verify that the ProgressCircle track within the Picker is using CanvasText color.

## 🧢 Your Project:

Adobe/Accessibility
